arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -484,7 +484,7 @@ RestStatus RestDocumentHandler::modifyDocument(bool isPatch) {
   }
 
   // find and load collection given by name or identifier
-  _activeTrx = createTransaction(cname, AccessMode::Type::WRITE);
+  _activeTrx = createTransaction(cname, AccessMode::Type::WRITE, opOptions);
 
```

Comment:
Again a couple if lines below we add IS_FOLLOWER_TRX twice

Reply:
Changed.